bin/prefix-vendor-namespace.php

```diff
@@ -141,3 +147,50 @@ function replace_in_json_file( string $file, string $namespace, string $new_name
 		)
 	);
 }
+
+function remove_file_autoloads( string $file, array $composer_json, string $package_name ) {
+	if ( ! file_exists( $file ) ) {
+		return;
+	}
+
+	$json = json_decode( file_get_contents( $file ), true );
+	if ( empty( $json['packages'] ) ) {
+		return;
+	}
+
+	$modified = false;
+	foreach( $json['packages'] as $key => $package ) {
+		if ( 0 !== strpos( $package['name'], $package_name ) ) {
+			continue;
+		}
+
+		if ( empty( $package['autoload']['files'] ) ) {
+			continue;
+		}
+
+		foreach( $package['autoload']['files'] as $autoload_file ) {
+
+			// Confirm we already include this autoload in the main composer file.
+			$filename = 'vendor/' . $package['name'] . '/' . $autoload_file;
```

Comment:
Minor: Let's use interpolation here instead of concatenation

```suggestion
			$filename = "vendor/{$package['name']}/{$autoload_file}";
```